openhands/events/observation/browse.py

```diff
@@ -38,6 +38,6 @@ def __str__(self) -> str:
             f'Last browser action: {self.last_browser_action}\n'
             f'Last browser action error: {self.last_browser_action_error}\n'
             f'Focused element bid: {self.focused_element_bid}\n'
-            f'axTree: {self.axtree_object}\n'
-            f'CONTENT: {self.content}\n'
+            # f'axTree: {self.axtree_object}\n' # too long
+            f'CONTENT: {self.content if len(self.content) < 1000 else self.content[:1000]}\n'
```

Comment:
Totally minor thing to note, we have some way to deal with too much content here https://github.com/All-Hands-AI/OpenHands/blob/5100d12cea2cd35c30a22e25fbac376b72ed0981/openhands/events/serialization/event.py#L89

(Also the method above it, event_to_memory() strips unnecessary stuff before sending to the LLM. But I think that one was before using browsergym and the browsing agent - and I'm not sure it ever worked with this particular observation.)

I wonder if we can reuse here the truncate method, or does this need to be exactly the first 1k?

Reply:
> I wonder if we can reuse here the truncate method, or does this need to be exactly the first 1k?

Sorry, actually this was my naive ad-hoc attempt to make the log on terminal visible because there're a lot of output from browser obs printed there. I'm not certain how to handle it correctly, I can revert this for a proper PR if others also feel necessary.

Reply:
I thought this method is only used to print to terminal, but seems like it is also used to send to LLM as well, which might be the reason why browsing agent performs kind of worse when testing.